WordPress/src/main/java/org/wordpress/android/ui/mysite/cards/post/PostCardBuilder.kt

```diff
@@ -1,35 +1,47 @@
 package org.wordpress.android.ui.mysite.cards.post
 
+import org.wordpress.android.R
 import org.wordpress.android.ui.mysite.MySiteCardAndItem.Card.PostCard
+import org.wordpress.android.ui.mysite.MySiteCardAndItem.Card.PostCard.PostCardDraftOrScheduled
+import org.wordpress.android.ui.mysite.MySiteCardAndItem.Card.PostCard.PostCardDraftOrScheduled.PostItem
 import org.wordpress.android.ui.mysite.MySiteCardAndItemBuilderParams.PostCardBuilderParams
+import org.wordpress.android.ui.mysite.cards.post.mockdata.MockedPostsData.Post
+import org.wordpress.android.ui.utils.UiString.UiStringRes
 import org.wordpress.android.ui.utils.UiString.UiStringText
 import javax.inject.Inject
 
 class PostCardBuilder @Inject constructor() {
-    fun build(params: PostCardBuilderParams): List<PostCard> {
-        val cards = mutableListOf<PostCard>()
-        params.mockedPostsData?.posts?.draft?.map {
-            cards.add(
-                    PostCard(
-                            title = UiStringText(DRAFT_TITLE),
-                            postTitle = UiStringText(it.title ?: NO_TITLE)
-                    )
-            )
-        }
-        params.mockedPostsData?.posts?.scheduled?.map {
-            cards.add(
-                    PostCard(
-                            title = UiStringText(SCHEDULED_TITLE),
-                            postTitle = UiStringText(it.title ?: NO_TITLE)
-                    )
-            )
-        }
-        return cards
+    fun build(params: PostCardBuilderParams) = mutableListOf<PostCard>().apply {
```

Comment:
Minor (🔍): Since this `build(...)` function will return an implicit `MutableList<PostCard>` due to the `mutableListOf<PostCard>()` construction, consider making the return type explicit and make the return type `List<PostCard>` so that it no longer becomes available to the client as a mutable list.

Reply:
Great, I am already seeing it as done through 34fb96ee037f454d7eab1e2ff6dc7b4e8f8061a0, thank you Ashita! 🙏